src/main/java/seedu/duke/exception/BaseException.java

```diff
@@ -0,0 +1,34 @@
+package seedu.duke.exception;
+
+@SuppressWarnings("serial")
+public abstract class BaseException extends Exception {
+    public static final String INDENT = "\t";
```

Comment:
Could consider extracting this out into the `Constants` class

Reply:
Thank you! Corrections has been pushed.